client/platform/desktop/frontend/components/Recent.vue

```diff
@@ -262,69 +335,67 @@ export default defineComponent({
             >
               Open images or video to get started
             </h2>
-            <div
-              v-for="recent in paginatedRecents"
-              :key="recent.id"
-              class="pa-1"
+            <v-data-table
+              dense
+              v-bind="{ headers: headers, items: filteredRecents }"
+              :footer-props="{ itemsPerPageOptions: [10, 30, -1] }"
+              no-data-text="No data loaded"
```

Comment:
We need to update the screenshot for desktop on <https://kitware.github.io/dive/Screenshots/#browse-files>